hawkular-alerts-engine/src/main/resources/org/hawkular/alerts/engine/rules/ConditionMatch.drl

```diff
@@ -293,16 +296,31 @@ rule AlertOnSatisfiedDampening
         $t  : Trigger( mode == Mode.FIRING, $tid : id )
         $d  : Dampening( triggerMode == Mode.FIRING, triggerId == $tid, satisfied == true )
     then
-        if (log != null && log.isDebugEnabled()) {
-            log.debug("AlertOnSatisfiedDampening! " + $d.log());
+        Event newEvent;

```

Comment:
Same of my previous comment, as we are still having some doubts in the api, I'm trying to not start the implementation yet.
Just to isolate, and not to bring yet ergonomics from implementation than might influence in the model.